src/Sylius/Component/Core/Cart/Context/ShopBasedCartContextInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Cart\Context;
+
+use Sylius\Component\Order\Context\CartContextInterface;
+use Sylius\Component\Order\Context\ResettingCartContextInterface;
+
+interface ShopBasedCartContextInterface extends CartContextInterface, ResettingCartContextInterface
+{
+
+}
```

Comment:
We're going to add the `reset` method on the `CartContextInterface` in the `2.0`, so I think we should remove this interface, and just declare `ResettingCartContextInterface` explicitly on `ShopBasedCartContext`.